shot/src/main/scala/com/karumi/shot/ShotPlugin.scala

```diff
@@ -61,28 +58,76 @@ class ShotPlugin extends Plugin[Project] {
   }
 
   private def addTasks(project: Project): Unit = {
+    val appExtension =
+      project.getExtensions.getByType[AppExtension](classOf[AppExtension])
+    val baseTask = project.getTasks.create(
+      Config.defaultTaskName,
+      classOf[ExecuteScreenshotTestsForEveryFlavor])
+    appExtension.getApplicationVariants.all { variant =>
+      val flavor = variant.getMergedFlavor
+      val completeAppId = flavor.getApplicationId + Option(
+        flavor.getApplicationIdSuffix).getOrElse("")
+      if (variant.getBuildType.getName != "release") {
```

Comment:
Interesting take, why do we prevent `release` builds to have a Shot task? Was this causing any issue or is it just to prevent people from running Shot in release builds? I think they can bypass this check by just renaming the release config on their side if that's the case.

Reply:
Just because there are no tasks to test any release build. If you check the ``androidTests`` tasks for your project you'll notice the release variant has no task associated.